services/dashboard/services/restapi.js

```diff
@@ -129,13 +129,16 @@ angular.module('dashboard')
             files, formFormNames, executorNum, args, onComplete);
         },
 
-        _submitApp: function(url, files, formFormNames, executorNum, args, onComplete) {
-          var params = '?executorNum=' + executorNum + '&args=' + encodeURIComponent(args);
+        _submitApp: function(url, files, fileFieldNames, executorNum, args, onComplete) {
           var upload = Upload.upload({
-            url: url + params,
+            url: url,
             method: 'POST',
             file: files,
-            fileFormDataName: formFormNames
+            fileFormDataName: fileFieldNames,
+            fields: {
+              "executornumber": executorNum,
```

Comment:
Maybe the correct name would be `executorCount`. And please update all occurrence, if the rename makes sense to you.